tests/conftest.py

```diff
@@ -22,13 +22,15 @@ def datagram_received(self, data, addr):
 @pytest.fixture
 def wait_for():
     async def _wait_for(
-        collected: List[str], *, count: int = 1, attempts: int = 5
+        collected: List[str], *, count: int = 1, attempts: int = 50
     ) -> None:
+        sleep = 0.0
         while attempts:
             if len(collected) == count:
                 break
 
             attempts -= 1
-            await asyncio.sleep(0.1)
+            await asyncio.sleep(sleep)
+            sleep = 0.01
```

Comment:
Can you please explain why we need that change?

Reply:
It is not so necessary, just small speed up of test suite, no need to wait for 0.1 coroutine may be finished earlier. 
I may revert this change if you want to

Reply:
Let's keep your change.